### PR TITLE
Carousel  - Namespace all swiper css overrides to stop clashes with other plugins

### LINKED ISSUE #20289: Carousel swiper css overrides other swiper based plugins
#### Steps to reproduce the issue

1. On a local test site install newspack blocks and jetpack
2. Turn on jetpack carousel and add a page with the Post Carousel block and a gallery
3. In frontend notice that the Post carousel overflows the page

Screen Shot 2021-07-07 at 4 37 38 PM

### PR DESCRIPTION
**Pull request by glendaviesnz**
Fixes: #20289

#### Changes proposed in this Pull Request:

* Adds a jp-carousel class to all swiper.js css overrides

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

- On a local test site install newspack blocks and jetpack
- Turn on jetpack carousel and add a page with the Post Carousel block and a gallery
- In frontend notice make sure that both the  Post carousel and the gallery carousel display as expected

Before:
Screen Shot 2021-07-07 at 4 37 38 PM

After:
Screen Shot 2021-07-07 at 5 15 51 PM

### CODE REVIEW

File: projects/plugins/jetpack/modules/carousel/jetpack-carousel.css [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/*
```
Comment (glendaviesnz) [inline projects/plugins/jetpack/modules/carousel/jetpack-carousel.css:1]: There is an issue in that Newspack Post carousel block doesn't properly override the next and previous buttons, so the default swiper css gets reapplied.

I believe it is a slower process to get Newspack changes through the ETK release process so have added a temporary fix for this in the carousel css - https://github.com/Automattic/jetpack/pull/20290/files#diff-72023849d8cb93014b6aafc178f8b48eadb46607ad5c9eb6b2a0c419deebf243R1

Our problem is if we do just reset this without the .wp-block-newspack-blocks-carousel selector we may break things for other plugins that use the slider defaults for this. 

The the other alternative is to customise projects/plugins/jetpack/modules/carousel/swiper-bundle.css  to not include it, but that means we need to remember to customise this file with every swiper library update.

**Review by aaronrobertshaw — APPROVED** [review]
The namespacing makes sense to me. As does the interim fix for the newspack blocks carousel prev/next buttons. Would it be worth adding a link to a newspack blocks issue within the appropriate comment?

#### Testing
1. Pulled latest for Newspack blocks and Jetpack and rebuilt
2. Confirmed I could replicate the issue
3. Applied this PR
4. Refreshed frontend and everything appeared correctly
5. For good measure reloaded the editor - no issues found.

LGTM 👍 

https://user-images.githubusercontent.com/60436221/124711133-08e49b00-df41-11eb-835d-afce50461886.mp4

**Review by ramonjd — APPROVED** [review]
✅ 

**Before**
Screen Shot 2021-07-07 at 5 10 29 pm

**After**
Screen Shot 2021-07-07 at 5 08 04 pm

**Review by sdixon194 — APPROVED** [review]
This works for me!

File: projects/plugins/jetpack/modules/carousel/jetpack-carousel.css (hunk 1 above)
Comment (kraftbj) [reply projects/plugins/jetpack/modules/carousel/jetpack-carousel.css:1]: Can you drop a link to the issue with the Newspack carousel so we can track when it is safe to remove the temporary fix?

Comment (glendaviesnz) [reply projects/plugins/jetpack/modules/carousel/jetpack-carousel.css:1]: https://github.com/Automattic/jetpack/issues/20305

**Comment by jeherve** [comment]
Cherry-picked to `jetpack/branch-9.9` in 1f773ce92801738adfa38e8d1e5a11e263f77c95

**Pull request merged**